homework/index.js

```diff
@@ -16,31 +37,139 @@
     xhr.send();
   }
 
-  function createAndAppend(name, parent, options = {}) {
-    const elem = document.createElement(name);
-    parent.appendChild(elem);
-    Object.keys(options).forEach(key => {
-      const value = options[key];
-      if (key === 'text') {
-        elem.textContent = value;
-      } else {
-        elem.setAttribute(key, value);
+  function Contributions(inText) {
+    const url = inText.innerHTML;
+    const infoContainer = document.getElementById('info_container');
+    const infoRight = createAndAppend('div', infoContainer, {
+      class: 'info_right',
+      text: 'Contributions:',
+    });
+    fetchJSON(url, (err, data) => {
+      if (err) {
+        renderError(err);
+        return;
       }
+      data.forEach(element => {
```

Comment:
Say what you mean:

```js
contributors.forEach(contributor => {
```

Reply:
I changed the names is the whole function. 
I should be more careful about that in my whole work.